frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/network-attachment-definition.tsx

```diff
@@ -0,0 +1,127 @@
+  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
+  classNames('col-lg-4', 'col-md-4', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
+  Kebab.columnClass,
+];
+
+const NetworkAttachmentDefinitionsHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Namespace',
+      sortField: 'metadata.namespace',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Type',
+      sortFunc: 'string',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[2] },
+    },
+    {
+      title: '',
+      props: { className: tableColumnClasses[3] },
+    },
+  ];
+};
+NetworkAttachmentDefinitionsHeader.displayName = 'NADTableHeader';
+
+const NetworkAttachmentDefinitionsRow: React.FC<NetworkAttachmentDefinitionsRowProps> = ({
+  obj,
+  index,
+  key,
+  style,
+}) => {
+  const config = getConfig(obj);
+  const name = getName(obj);
+  const namespace = getNamespace(obj);
+  const uid = getUID(obj);
+
+  return (
+    <TableRow id={uid} index={index} trKey={key} style={style}>
+      <TableData className={classNames(tableColumnClasses[0], 'co-break-word')}>
+        <ResourceLink
+          kind={NetworkAttachmentDefinitionModel.kind}
+          name={name}
+          namespace={namespace}
+        />
+      </TableData>
+      <TableData className={classNames(tableColumnClasses[1], 'co-break-word')}>
+        <ResourceLink kind={NamespaceModel.kind} name={namespace} title={namespace} />
+      </TableData>
+      <TableData className={tableColumnClasses[2]}>
+        {config.type || <span className="text-secondary">Not available</span>}
+      </TableData>
+      <TableData className={tableColumnClasses[3]}>
+        <ResourceKebab
+          actions={menuActions}
+          kind={NetworkAttachmentDefinitionReference}
+          resource={obj}
+        />
+      </TableData>
+    </TableRow>
+  );
+};
+NetworkAttachmentDefinitionsRow.displayName = 'NetworkAttachmentDefinitionsRow';
+
+type NetworkAttachmentDefinitionsRowProps = {
+  obj: NetworkAttachmentDefinitionKind;
+  index: number;
+  key?: string;
+  style: object;
+  namespace: string;
+};
+
+export const NetworkAttachmentDefinitionsList: React.SFC = (props) => (
+  <Table
+    {...props}
+    aria-label={NetworkAttachmentDefinitionModel.labelPlural}
+    Header={NetworkAttachmentDefinitionsHeader}
+    Row={NetworkAttachmentDefinitionsRow}
+    virtualize
+  />
+);
+NetworkAttachmentDefinitionsList.displayName = 'NetworkAttachmentDefinitionsList';
+
+export const NetworkAttachmentDefinitionsPage: React.SFC<NetworkAttachmentDefinitionsPageProps> = (
+  props,
+) => (
+  <ListPage
+    {...props}
+    title={NetworkAttachmentDefinitionModel.labelPlural}
+    kind={NetworkAttachmentDefinitionReference}
```

Comment:
why not use `NetworkAttachmentDefinitionModel.kind` ? + in all the other places

Reply:
It was an oversight. I created the reference when I first started and never fixed it. Thanks for catching it. I believe I've updated it in all of the relevant places.